src/main/java/dk/bec/polonez/reservationsystem/controller/OfferController.java

```diff
@@ -44,8 +44,8 @@ public OfferDto deleteOffer(@PathVariable long id) {
         return offerService.deleteOffer(id);
     }
 
-    @PostMapping("feature")
-    public ResponseOfferFeatureDto addFeatureToOffer(@RequestParam long featureId, @RequestParam long offerId) {
+    @PostMapping("{id}/feature")
```

Comment:
**advice** Should be ```{id}/features``` according to REST convention because POST is always about adding new resource to a collection of resources (here: collections of features), that's why plural is needed.